doc/whats_new.rst

```diff
@@ -43,6 +43,9 @@ Changelog
   returned for cases when there should be no valid matches, by `George Dang`_
   in :gh:`739`
 
+- Added check for invalid Axes object in :func:`~hnn_core/viz/plot_cells` 
+  function, by `Abdul Samad Siddiqui`_ in :gh:`744`
```

Comment:
Nitpick, but we should probably be consistent with punctuation (which none of us have been good about so far).

```suggestion
  function, by `Abdul Samad Siddiqui`_ in :gh:`744`.
```